pages/games/[gameType].tsx

```diff
@@ -511,7 +499,8 @@ const Game: NextPage<IGamePageProps> = ({ gameType, version }) => {
     }
     console.log('polling for opponent address, gameId=', gameId);
     let players: [string, string] = await gameApi.getPlayers(
-      await getArbiter(),
+      // const contract = await getArbiter(); //TEMP
```

Comment:
If this is a temporary solution, when should it be changed, what do we need before it can be changed - is it some answer from Wagmi team?